src/views/layouts/main.hbs

```diff
@@ -1,3 +1,4 @@
+{{>navbar}}
```

Comment:
lets keep the file naming consistent with kabab case ( nav-bar instead of navbar )